widgetIDESupport/index.js

```diff
@@ -684,7 +683,7 @@ if (TW.IDE && (typeof TW.IDE.Widget == 'function')) {
                 }
             },
 
-            [versionSymbol]: 3
+            [versionSymbol]: 4
```

Comment:
Can we by any chance extract this number into a "global" constant?
I see it's used in three places, and the one that kinda concerns me is the if statement where the "4" is hidden.